shacl12-node-expr/index.html

```diff
@@ -859,14 +859,14 @@ <h3>List Parameter Functions</h3>
 	sh:values <b>[
 		ex:coalesce (
 			[
-				# This is a path expression that is expected to return zero or one values
-				shnex:path ex:fullName ;
+				# This is a path values expression that is expected to return zero or one values
+				shnex:pathValues ex:fullName ;
```

Comment:
I think this may be the root of the difference between `pathValues` and `pathValue`. "zero or one values" leads to the troublesomely indeterminate expansion of "zero values or one value". `zero pathValues` reads better than `zero pathValue`, while `one pathValue` reads better than `one pathValues`. (`no pathValue` and `no pathValues` both read OK.)

It's too bad parentheses are problematic in predicate identifiers, e.g., `shnex:pathValue(s)` and "zero or one value(s)". I think these might be acceptable to all, and perhaps the problems are small enough to likewise accept.

Similar discussion applies to `focus node`, `focus nodes`, and `focus node(s)`.

Reply:
Fair enough.